### PR TITLE
Updated documentation 

### PR DESCRIPTION
**Pull request by renishnair-viper**
Updated documentation with the following points.

* OpenVPN TAP adapters for Win 10 and above have been updated and are now WHQL certified for improved compatibility and stability. To update the TAP adapters, simply upgrade the OpenVPN.Binaries.Scramble nuGet package to version 2.5.3
* Added support to enable threat protection while the VPN connection is active
* Domain-based split tunneling can now be operational at the same time as an active kill switch
* Added token-based support, allowing users to log in using access and refresh tokens
* Added support for adding and fetching account metadata
* Added new methods for stopping and uninstalling the split tunnel driver service.
* Removed the logging of DNS requests unless Debug Mode is enabled.

Bug Fixes:

* Fixed an issue where some network filters failed to clear correctly following an abrupt SDK termination
* Fixed a minor issue when disconnecting the SDK when domain based split tunneling is active

Following points are covered for ConsumerVPN app.

* Added toggle in Connection Settings for enabling/disabling threat protection.
* Updated OpenVPN.Binaries.Scramble nuGet package to version 2.5.3
* Updated VpnSDK nuGet package to version 2.2.32

### CODE REVIEW

**Review by rastapopoulos1 — APPROVED** [review]

File: v2/articles/sdk-changelog.html [hunk 1]
```diff
@@ -132,71 +132,86 @@
...
 <h1 id="vpnsdk-changelog">VpnSDK Changelog</h1>
 
-<h2 id="v2231">v2.2.31</h2>
+<h2 id="v2232">v2.2.32</h2>
 <h3 id="features">Features</h3>
 <ol>
+<li>OpenVPN TAP adapters for Win 10 and above have been updated and are now WHQL certified for improved compatibility and stability. 
+To update the TAP adapters, simply upgrade the OpenVPN.Binaries.Scramble nuGet package to version 2.5.3.</li>
+<li>Added support to enable threat protection while the VPN connection is active.</li>
+<li>Domain-based split tunneling can now be operational at the same time as an active kill switch.</li>
```
Comment (vimalraj-adikesavan) [inline v2/articles/sdk-changelog.html:141]: This statement is not clear.

File: v2/articles/sdk-changelog.html [hunk 2]
```diff
@@ -132,71 +132,86 @@
...
+<li>Added token-based support, allowing users to log in using access and refresh tokens.</li>
+<li>Added support for adding and fetching account metadata.</li>
+</ol>
+<h3 id="bug-fixes">Bug fixes</h3>
+<ol>
+<li>Fixed an issue where some network filters failed to clear correctly following an abrupt SDK termination.</li>
+<li>Fixed a minor issue when disconnecting the SDK when domain based split tunneling is active.</li>
```
Comment (vimalraj-adikesavan) [inline v2/articles/sdk-changelog.html:150]: This statement is not clear.

File: v2/articles/sdk-changelog.html (hunk 1 above)
Comment (renishnair-viper) [reply v2/articles/sdk-changelog.html:141]: Discussed so closing this conversation

File: v2/articles/sdk-changelog.html (hunk 2 above)
Comment (renishnair-viper) [reply v2/articles/sdk-changelog.html:150]: Discussed so closing this conversation

**Review by vimalraj-adikesavan — APPROVED** [review]

**Review by rastapopoulos1 — APPROVED** [review]

**Review by rastapopoulos1 — APPROVED** [review]

**Review by vimalraj-adikesavan — APPROVED** [review]

**Pull request merged**